modules/mtbseq/tbbwa.nf

```diff
@@ -3,6 +3,8 @@ process TBBWA {
     label 'process_medium'
     publishDir params.results_dir, mode: params.save_mode, enabled: params.should_publish
 
+    conda "bioconda::mtbseq=1.1.0"
+    container "${'quay.io/biocontainers/mtbseq:1.1.0--hdfd78af_0'}"
```

Comment:
The container names had a typo initially, they were named as `bquay.io` therefore the docker profile was failing.